### PR TITLE
[Outlook] (format) Reformat table to improve readability

### PR DESCRIPTION
**Pull request by samantharamon**
Fix table formatting to improve readability.

Internat work item reference number: 5731758

### CODE REVIEW

File: docs/outlook/autolaunch.md [hunk 1]
```diff
@@ -17,24 +17,24 @@ By the end of this walkthrough, you'll have an add-in that runs whenever a new i
...
+|Event|Description|Supported minimum requirement set and clients|
+|---|---|---|
+|`OnNewMessageCompose`|On composing a new message (includes reply, reply all, and forward) but not on editing, for example, a draft.|[1.10](../reference/objectmodel/requirement-set-1.10/outlook-requirement-set-1.10.md)<br><br>Windows, web browser, new Mac UI preview|
+|`OnNewAppointmentOrganizer`|On creating a new appointment but not on editing an existing one.|[1.10](../reference/objectmodel/requirement-set-1.10/outlook-requirement-set-1.10.md)<br><br>Windows, web browser, new Mac UI preview|
+|`OnMessageAttachmentsChanged`|On adding or removing attachments while composing a message.<br><br>Event information listed in [AttachmentsChangedEventArgs](/javascript/api/outlook/office.attachmentschangedeventargs?view=outlook-js-1.11&preserve-view=true).|[1.11](../reference/objectmodel/requirement-set-1.11/outlook-requirement-set-1.11.md)<br><br>Windows, web browser|
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/autolaunch.md:29]: I'd recommend something like "Event-specific data object:" and bulleting clients list
```suggestion
|`OnMessageAttachmentsChanged`|On adding or removing attachments while composing a message.<br><br>Event-specific data object: [AttachmentsChangedEventArgs](/javascript/api/outlook/office.attachmentschangedeventargs?view=outlook-js-1.11&preserve-view=true)|[1.11](../reference/objectmodel/requirement-set-1.11/outlook-requirement-set-1.11.md)<br><br>- Windows<br>- web browser|
```

File: docs/outlook/autolaunch.md [hunk 2]
```diff
@@ -17,24 +17,24 @@ By the end of this walkthrough, you'll have an add-in that runs whenever a new i
...
-|`OnInfoBarDismissClicked`|On dismissing a notification while composing a message or appointment item. Only the add-in that added the notification will be notified.|[InfobarClickedEventArgs](/javascript/api/outlook/office.infobarclickedeventargs?view=outlook-js-1.11&preserve-view=true)|[1.11](../reference/objectmodel/requirement-set-1.11/outlook-requirement-set-1.11.md)|Windows, web browser|
-|`OnMessageSend`|On sending a message item. To learn more, refer to the [Smart Alerts walkthrough](smart-alerts-onmessagesend-walkthrough.md).|Not applicable|[Preview](../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md)|Windows|
-|`OnAppointmentSend`|On sending an appointment item. To learn more, refer to the [Smart Alerts walkthrough](smart-alerts-onmessagesend-walkthrough.md).|Not applicable|[Preview](../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md)|Windows|
+|Event|Description|Supported minimum requirement set and clients|
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/autolaunch.md:25]:
```suggestion
|Event|Description|Minimum requirement set and supported clients|
```

**Review by ElizabethSamuel-MSFT — COMMENTED** [review]
This looks great. Just have a couple of suggestions.

File: docs/outlook/autolaunch.md [hunk 3]
```diff
@@ -17,24 +17,24 @@ By the end of this walkthrough, you'll have an add-in that runs whenever a new i
...
+|`OnAppointmentRecurrenceChanged`|On adding, changing, or removing the recurrence details while composing an appointment. If the date/time is changed, the `OnAppointmentTimeChanged` event will also be fired.<br><br>Event-specific data object: [RecurrenceChangedEventArgs](/javascript/api/outlook/office.recurrencechangedeventargs?view=outlook-js-1.11&preserve-view=true).|[1.11](../reference/objectmodel/requirement-set-1.11/outlook-requirement-set-1.11.md)<br><br>- Windows<br>- Web browser|
+|`OnInfoBarDismissClicked`|On dismissing a notification while composing a message or appointment item. Only the add-in that added the notification will be notified.<br><br>Event-specific data object: [InfobarClickedEventArgs](/javascript/api/outlook/office.infobarclickedeventargs?view=outlook-js-1.11&preserve-view=true).|[1.11](../reference/objectmodel/requirement-set-1.11/outlook-requirement-set-1.11.md)<br><br>- Windows<br>- Web browser|
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/autolaunch.md:35]: Minor tweak that removes the period from the end of the "Event-specific..." link
```suggestion
|`OnAppointmentAttachmentsChanged`|On adding or removing attachments while composing an appointment.<br><br>Event-specific data object: [AttachmentsChangedEventArgs](/javascript/api/outlook/office.attachmentschangedeventargs?view=outlook-js-1.11&preserve-view=true)|[1.11](../reference/objectmodel/requirement-set-1.11/outlook-requirement-set-1.11.md)<br><br>- Windows<br>- Web browser|
|`OnMessageRecipientsChanged`|On adding or removing recipients while composing a message.<br><br>Event-specific data object: [RecipientsChangedEventArgs](/javascript/api/outlook/office.recipientschangedeventargs?view=outlook-js-1.11&preserve-view=true).|[1.11](../reference/objectmodel/requirement-set-1.11/outlook-requirement-set-1.11.md)<br><br>- Windows<br>- Web browser|
|`OnAppointmentAttendeesChanged`|On adding or removing attendees while composing an appointment.<br><br>Event-specific data object: [RecipientsChangedEventArgs](/javascript/api/outlook/office.recipientschangedeventargs?view=outlook-js-1.11&preserve-view=true)|[1.11](../reference/objectmodel/requirement-set-1.11/outlook-requirement-set-1.11.md)<br><br>- Windows<br>- Web browser|
|`OnAppointmentTimeChanged`|On changing date/time while composing an appointment.<br><br>Event-specific data object: [AppointmentTimeChangedEventArgs](/javascript/api/outlook/office.appointmenttimechangedeventargs?view=outlook-js-1.11&preserve-view=true)|[1.11](../reference/objectmodel/requirement-set-1.11/outlook-requirement-set-1.11.md)<br><br>- Windows<br>- Web browser|
|`OnAppointmentRecurrenceChanged`|On adding, changing, or removing the recurrence details while composing an appointment. If the date/time is changed, the `OnAppointmentTimeChanged` event will also be fired.<br><br>Event-specific data object: [RecurrenceChangedEventArgs](/javascript/api/outlook/office.recurrencechangedeventargs?view=outlook-js-1.11&preserve-view=true)|[1.11](../reference/objectmodel/requirement-set-1.11/outlook-requirement-set-1.11.md)<br><br>- Windows<br>- Web browser|
|`OnInfoBarDismissClicked`|On dismissing a notification while composing a message or appointment item. Only the add-in that added the notification will be notified.<br><br>Event-specific data object: [InfobarClickedEventArgs](/javascript/api/outlook/office.infobarclickedeventargs?view=outlook-js-1.11&preserve-view=true)|[1.11](../reference/objectmodel/requirement-set-1.11/outlook-requirement-set-1.11.md)<br><br>- Windows<br>- Web browser|
```

File: docs/outlook/autolaunch.md [hunk 4]
```diff
@@ -17,24 +17,24 @@ By the end of this walkthrough, you'll have an add-in that runs whenever a new i
...
+|`OnMessageSend`|On sending a message item. To learn more, refer to the [Smart Alerts walkthrough](smart-alerts-onmessagesend-walkthrough.md).|[Preview](../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md)<br><br>Windows|
+|`OnAppointmentSend`|On sending an appointment item. To learn more, refer to the [Smart Alerts walkthrough](smart-alerts-onmessagesend-walkthrough.md).|[Preview](../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md)<br><br>Windows|
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/autolaunch.md:37]: Consider including bullet for consistent look and feel
```suggestion
|`OnMessageSend`|On sending a message item. To learn more, refer to the [Smart Alerts walkthrough](smart-alerts-onmessagesend-walkthrough.md).|[Preview](../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md)<br><br>- Windows|
|`OnAppointmentSend`|On sending an appointment item. To learn more, refer to the [Smart Alerts walkthrough](smart-alerts-onmessagesend-walkthrough.md).|[Preview](../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md)<br><br>- Windows|
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Awesome! Just a couple of minor suggestions.

File: docs/outlook/autolaunch.md (hunk 4 above)
Comment (samantharamon) [reply docs/outlook/autolaunch.md:37]: Thanks for catching those.

File: docs/outlook/autolaunch.md (hunk 3 above)
Comment (samantharamon) [reply docs/outlook/autolaunch.md]: Periods have now been removed from the event-specific object links.

**Pull request merged**